optuna/cli.py

```diff
@@ -282,27 +290,15 @@ def take_action(self, parsed_args):
             )
 
 
-_COMMANDS = {
-    "create-study": _CreateStudy,
-    "delete-study": _DeleteStudy,
-    "study set-user-attr": _StudySetUserAttribute,
-    "studies": _Studies,
-    "dashboard": _Dashboard,
-    "study optimize": _StudyOptimize,
-    "storage upgrade": _StorageUpgrade,
-}
-
-
 class _OptunaApp(App):
     def __init__(self):
         # type: () -> None
 
-        command_manager = CommandManager("optuna.command")
         super(_OptunaApp, self).__init__(
-            description="", version=optuna.__version__, command_manager=command_manager
+            description="",
+            version=optuna.__version__,
+            command_manager=CommandManager("optuna.command"),
         )
-        for name, cls in _COMMANDS.items():
-            command_manager.add_command(name, cls)
```

Comment:
By https://github.com/optuna/optuna/pull/60 , the registration was changed from setup.py to cli.py.

But I reverted it in this PR because:
1. #60 does not follow [the standard way](https://docs.openstack.org/cliff/latest/user/demoapp.html), and difficult to understand the structure.
2. I could not generate sphinx docs automatically.